packages/wordpress-block-editor-config/README.md

```diff
@@ -0,0 +1,24 @@
+# WordPress Block Editor Config
+
+Package to share common settings for WordPress specific Block Editors.
+
+## Installation
+
+Install the module
+
+```bash
+npm install @wordpress/wordpress-block-editor --save
+```
+
+_This package assumes that your code will run in an **ES2015+** environment. If you're using an environment that has limited or no support for ES2015+ such as lower versions of IE then using [core-js](https://github.com/zloirock/core-js) or [@babel/polyfill](https://babeljs.io/docs/en/next/babel-polyfill) will add support for these methods. Learn more about it in [Babel docs](https://babeljs.io/docs/en/next/caveats)._
+
+## API
+
+<!-- START TOKEN(Autogenerated API docs) -->
+
+Nothing to document.
```

Comment:
Note that autodoc skips over our example due to the only entry being prefixed with __experimental.